pkgs/development/web/flyctl/default.nix

```diff
@@ -35,6 +35,12 @@ buildGoModule rec {
     go test ./... -ldflags="-X 'github.com/superfly/flyctl/internal/buildinfo.buildDate=1970-01-01T00:00:00+0000'"
   '';
 
+  doInstallCheck = true;
+
+  installCheckPhase = ''
+    $out/bin/flyctl version | grep ${version} > /dev/null
```

Comment:
hi @techknowlogick 

we now have `testVersion` for this

https://nixos.org/manual/nixpkgs/unstable/#chap-testers

Reply:
thanks for feedback :) I've updated PR to switch to that.